README.md

```diff
@@ -9,72 +9,40 @@ The application has only been tested with python 2.7.10. It will most likely wor
 `virtualenv` is not a requirement but it will help you to install the application.
 
 1. Download the repository from [github](https://github.com/NBISweden/swefreq)
-2. Create a file named `secrets.py` and place it in the same directory as `route.py`
-3. MySql. Only tested with 5.6, but should work with version 4 and above
+2. Rename the file `settings_sample.json` into `settings.json` and edit all
+   the values.
+3. Install MySql. Only tested with 5.6, but should work with version 4 and above
 
-The file `secrets.py` should define the following python variables:
 
-	# Google app-keys
-	googleKey = 
-	googleSecret = 
+4. If you have `virtualenv` and `pip` installed then you can do the following
+   to install the required python packages. Locate the `requirements.txt` file
+   in the `swefreq` repository.
 
-	# URI that google will redirect login to
-	redirect_uri = 
 
-	# MySql settings
-	mysqlHost = 
-	mysqlSchema = 
-	mysqlUser = 
-	mysqlPasswd = 
-	
-	# Mongodb settings
-	mongodbhost = 
-	monogdbport = 
-	
-	# URL to the ExAC server
-	ExAC_server = 
-	
-	# port to bind application
-	appPort =
-	
-	# ssl certificate files
-	cert = 
-	key = 
+    source /path/to/bin/activate             # activate your virtualenv
+    pip install -r /path/to/requrements.txt  # install the required python packages
 
-	# e-mail config
-	MAIL_SERVER = 
-	FROM_ADDRESS = 
-	REPLY_TO_ADDRESS = 
-	ADMIN_ADDRESS = 
 
+5. Create the MySql database, user and tables with the following command:
 
 
+       mysql -u root -p < ./sql/swefreq.sql
 
-If you have `virtualenv` and `pip` installed then you can do the following to install the required python packages. Locate the `requirements.txt` file in the `swefreq` repository.
 
+   To experience the full site you need to manually add a dataset and a user
+   to the database. Log into the mysql console and enter something like the
+   following:
 
-	source /path/to/bin/activate             # activate your virtualenv
-	pip install -r /path/to/requrements.txt  # install the required python packages
 
+       USE swefreq;
+       INSERT INTO dataset (dataset_pk, name) VALUES (1, "SweGen");
+       INSERT INTO user (user_pk,name, email) VALUES (1, "test", "myemail@google.com");
```

Comment:
I disagree with this one. Because currently the login is dependent on specifically googles OAuth, so it's there because of that. We should probably document setting up an app in google for it though.